mysql-test/t/percona_encrypt_tmp_files.test

```diff
@@ -1,5 +1,148 @@
 --source include/have_innodb.inc
 --source include/have_myisam.inc
+--source include/have_grep.inc
+--source include/have_debug_sync.inc
```

Comment:
We would still test the feature code under the release build too, even if we don't have any sync guarantees there (and cannot grep for plaintext). Consider splitting a testcase into two, maybe with a shared include file